### ISSUE TITLE
Crash on start up...

### ISSUE THREAD

**Issue by CupOJo1** [MoonflowerTeam/pollen#42]
### What version are you seeing the problem on?

1.18.2

### Describe the issue

```markdown
1. I assume its a mod conflict so not really sure what the first step would be...
```

### latest.log

```shell
https://gist.github.com/CupOJo1/1e194fe83b7c929d18de1a2da42170e1
```

**Comment by maximogta** [comment]
Same crash here, on world start.
https://pastebin.com/te1EM8tN

**Comment by Aquonyx** [comment]
I am having the same problem, it seems to be an issue with the data pack it's creating. I think the exact point it happens is when it says "Cannot invoke "gg.moonflower.pollen.api.resource.modifier.ResourceModifierManager$SidedReloader.getCompleteFuture()" because "gg.moonflower.pollen.api.resource.modifier.ResourceModifierManager.serverReloader" is null"

**Comment by Ocelot5836** [comment]
What Pollen version are you using?

**Comment by jacksonhardaway** [comment]
Released fix in Pollen 1.4.2
